client/homebrew/brewRenderer/brewRenderer.less

```diff
@@ -1,10 +1,13 @@
 @import (multiple, less) 'shared/naturalcrit/styles/reset.less';
 
 .brewRenderer {
-	will-change : transform;
-	overflow-y  : scroll;
+	will-change   : transform;
+	overflow-y    : scroll;
+	padding-block : 30px;

```

Comment:
Print offset is caused by this `padding-block`.

Reply:
@Gazook89 see, i knew this padding would come back to bite us in the ass.